test/e2e/snaps/test-snap-networkaccess.spec.js

```diff
@@ -0,0 +1,109 @@
+  it('test the network-access endowment', async function () {
+    const ganacheOptions = {
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: 25000000000000000000,
+        },
+      ],
+    };
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions,
+        failOnConsoleError: false,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        // enter pw into extension
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // navigate to test snaps page and connect to dialog snap
+        await driver.openNewPage(TEST_SNAPS_WEBSITE_URL);
+        await driver.delay(1000);
+        const dialogButton = await driver.findElement(
+          '#connectNetworkAccessSnap',
+        );
+        await driver.scrollToElement(dialogButton);
+        await driver.delay(1000);
+        await driver.clickElement('#connectNetworkAccessSnap');
+        await driver.delay(1000);
+
+        // switch to metamask extension and click connect
+        let windowHandles = await driver.waitUntilXWindowHandles(
+          3,
+          1000,
+          10000,
+        );
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({
+          text: 'Connect',
+          tag: 'button',
+        });
+
+        await driver.waitForSelector({ text: 'Approve & install' });
+
+        await driver.clickElement({
+          text: 'Approve & install',
+          tag: 'button',
+        });
+
+        await driver.waitForSelector({ text: 'Ok' });
+
+        await driver.clickElement({
+          text: 'Ok',
+          tag: 'button',
+        });
+
+        // switch to test snaps tab
+        await driver.switchToWindowWithTitle('Test Snaps', windowHandles);
+
+        // wait for npm installation success
+        await driver.waitForSelector({
+          css: '#connectNetworkAccessSnap',
+          text: 'Reconnect to networkAccess Snap',
+        });
+
+        // click on alert dialog
+        await driver.clickElement('#sendNetworkAccessTest');
+        await driver.delay(500);
+
+        // switch to dialog popup
+        windowHandles = await driver.waitUntilXWindowHandles(3, 1000, 10000);
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.delay(500);
+
+        // check dialog contents
+        const result = await driver.findElement('.snap-ui-renderer__panel');
+        await driver.scrollToElement(result);
+        await driver.delay(500);
+        assert.equal(
+          await result.getText(),
+          `{ "result": "FETCHED_SUCCESSFULLY" }`,
```

Comment:
```suggestion
          'FETCHED_SUCCESSFULLY',
```
Looks like this changed since writing the E2E.